Source/ASDisplayNode.h

```diff
@@ -621,29 +621,32 @@ extern NSInteger const ASDefaultDrawingPriority;
  */
 - (void)layoutIfNeeded;
 
-@property (nonatomic, strong, nullable) id contents;                           // default=nil
+@property (nonatomic, assign)           CGRect frame;                          // default=CGRectZero
```

Comment:
We should definitely do additional cleanup in this header. However, I made a few improvements (no breaking changes of course) so that the most commonly used properties are more towards the top and more logically grouped.

Because this is still pretty far from ideal, let's not worry too much about optimizing it in this patch - but at least it will be slightly better until we get around to it.